app/models/attribute.py

```diff
@@ -0,0 +1,27 @@
+from .. import db
+
+users_association_table = db.Table(
+    'users_association', db.Model.metadata,
```

Comment:
I don't know if we are going to have any other many to many relationships with user, but should we name this table something like `user_tag_associations`?